runtime.go

```diff
@@ -483,6 +483,70 @@ func RenderAttributes(ctx context.Context, w io.Writer, attributes Attributer) (
 					return err
 				}
 			}
+		// Integer types
```

Comment:
Comments in templ always end with a full stop, but there's no need for these comments, remove them.